frontend/src/app/testQueue/TestCardForm/diseaseSpecificComponents/SyphilisAoEForm.tsx

```diff
@@ -0,0 +1,189 @@
+            options={GENDER_IDENTITY_VALUES}
+            onChange={onSexualPartnerGenderChange}
+            initialSelectedValues={selectedGenders}
+            label={"What is the gender of their sexual partners?"}
+            required
+            validationStatus={
+              showGenderOfSexualPartnersError ? "error" : undefined
+            }
+            errorMessage={
+              showGenderOfSexualPartnersError &&
+              "Please answer this required question."
+            }
+          ></MultiSelect>
+        </div>
+      </div>
+      <div className="grid-row">
+        <div className="grid-col-auto">
+          <RadioGroup
+            legend="Is the patient pregnant?"
+            name={`pregnancy-${testOrder.internalId}`}
+            onChange={onPregnancyChange}
+            buttons={pregnancyResponses}
+            selectedRadio={responses.pregnancy}
+            required
+            validationStatus={showPregnancyError ? "error" : undefined}
+            errorMessage={
+              showPregnancyError && "Please answer this required question."
+            }
+          />
+        </div>
+      </div>
+      <div className="grid-row">
+        <div className="grid-col-auto">
+          <YesNoRadioGroup
+            name={`has-any-symptoms-${testOrder.internalId}`}
+            legend="Is the patient currently experiencing any symptoms?"
+            value={hasSymptoms}
+            required
+            onChange={onHasAnySymptomsChange}
+            validationStatus={showSymptomsError ? "error" : undefined}
+            errorMessage={
+              showSymptomsError && "Please answer this required question."
+            }
+          />
+        </div>
+      </div>
+      {hasSymptoms === "YES" && (
+        <div className={"grid-row grid-gap width-full flex-start"}>
+          <div className={`flex-${CHECKBOX_COLS_TO_DISPLAY - 1}`}>
+            <Checkboxes
+              boxes={syphilisSymptomDefinitions.map(({ label, value }) => ({
+                label,
+                value,
+                checked: symptoms[value],
+              }))}
+              legend="Select any symptoms the patient is experiencing"
+              name={`symptoms-${testOrder.internalId}`}
+              onChange={(e) => onSymptomsChange(e, symptoms)}
+              numColumnsToDisplay={CHECKBOX_COLS_TO_DISPLAY}
+              validationStatus={showSymptomSelectionError ? "error" : undefined}
+              errorMessage={
+                showSymptomSelectionError
+                  ? "Please answer this required question."
+                  : ""
+              }
+            />
+          </div>
+
+          <TextInput
+            data-testid="symptom-date"
+            name={`symptom-date-${testOrder.internalId}`}
+            type="date"
+            required
+            label="Date of symptom onset"
+            aria-label="Symptom onset date"
+            min={formatDate(new Date("Jan 1, 2020"))}
```

Comment:
should this be more flexible? I actually have no idea 🤔

Reply:
I think we do something similar for the COVID one (https://github.com/CDCgov/prime-simplereport/blob/main/frontend/src/app/testQueue/TestCardForm/diseaseSpecificComponents/CovidAoEForm.tsx) and I think functionally no one's going to backdate before the beginning of the pandemic. Think it's fine to leave it as is